features/runtime_versions.feature

```diff
@@ -2,24 +2,42 @@ Feature: Runtime versions are included in all requests
 
     Scenario Outline: Uncaught exception contains runtime versions for Android
         When I launch an <platform> app which has an uncaught exception
+        And I wait for 1 second
```

Comment:
Is there any way we can avoid this?

Reply:
Not from what I can tell, with no wait it fails the iOS tests.  However these should be removed fairly promptly as this repo is updated to MazeRunner 2.  So not ideal, but temporary

Reply:
With maze 2 we can